python/bin/dataproc_template_functions.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Check if an environment variable is set,
+# takes the name of the environment variable as an argument
+check_required_envvar() {
+  name=$1        # We pass in the variable to check as a string
+  value=${!name} # Indirect expansion to get the value
+  if [ -z "${value}" ]
+  then
+    echo "Required environment variable ${name} is missing"
+    Help && exit 1
+  else
+    echo "${name}=${value}"
+  fi
+}
+
+#Mandatory vs optional  specify
+Help() {
+  # Display Help
+  help_text=$(cat << EndOfMessage
+    Usage:
+
+    # Environment variables
+    export GCP_PROJECT=projectId
+    export REGION=us-west1
```

Comment:
I believe we should not have actual values in the example

Reply:
Do you mean keeping the values empty for the example?

Reply:
(this is also nitpicking)
But for example in `export GCP_PROJECT=projectId` no real project id is provided. 
We could have `export REGION=region`

Reply:
Changed the example to "region" :)